src/routes/create-klage.tsx

```diff
@@ -23,25 +23,31 @@ const CreateKlage = () => {
         }
 
         const query = queryString.parse(search);
-        const temaKey = ensureStringIsTema(getQueryValue(query.tema));
 
+        const temaKey = ensureStringIsTema(getQueryValue(query.tema));
         if (temaKey === null) {
             setError(`Ugyldig tema "${query.tema}".`);
             return;
         }
 
-        const fullmaktsgiver = getQueryValue(query.fullmaktsgiver);
+        const titleKey = getQueryValue(query.tittel);
+        if (titleKey === null || !TITLES.validateTitleKey(titleKey)) {
```

Comment:
Av det jeg har testet så vil `this.titles.has(null);` gi `false` - kanskje vi kunne bare skrevet `if (!TITLES.validateTitleKey(titleKey))` ?

Reply:
TypeScript godtar ikke noe annet enn `string` som argument til `Map<string, string>`.
Da må vi i så fall gjøre om fra `Map<string, string>` til `Map<string | null, string>`.

Oppdaterte logikken ift. at `titleKey` blir optional og i samsvar med forslaget ditt :)